docs/edgeql/primer.rst

```diff
@@ -0,0 +1,774 @@
+.. _ref_eql_primer:
+
+Primer
+======
+
+This page is indended as a rapid-fire overview of EdgeQL so you can hit
+the ground running with EdgeDB. Refer to the linked pages for more in-depth
+documentation.
+
+.. note::
+
+  Sample code is also shown for the :ref:`TypeScript query builder
```

Comment:
Can we add yet another explanation here, that EdgeDB is a database and EdgeQL is its query that replaces SQL. But we also have a great TypeScript library that we showcase here because it's the best thing for you if you build a TS app.